hw-model/src/model_emulated.rs

```diff
@@ -91,4 +91,8 @@ impl crate::HwModel for ModelEmulated {
 
         &mut self.output
     }
+
+    fn tracing_hint(&mut self, _enable: bool) {
```

Comment:
These tracing changes aren't mentioned in the PR description. Perhaps you intend this to be a separate PR?

Reply:
Sure.